lib/ws-server.js

```diff
@@ -0,0 +1,68 @@
+module.exports = WebSocketServer;
+
+var assert = require('assert');
+var debug = require('./debug')('process:' + process.pid);
+var message = require('./message');
+var util = require('util');
+var url = require('url');
+var Server = require('ws').Server;
+var WebSocketChannel = require('./ws-channel');
+
+
+function WebSocketServer(options, callback) {
```

Comment:
What is `callback` for? I don't see it used here, but maybe its used in the tests for something.